bayesian/utils.py

```diff
@@ -64,17 +64,19 @@ def get_user_email(user_profile):
 
 
 def create_component_bookkeeping(ecosystem, packages_list,
-                                 user_id, user_agent, manifest_hash, request_id):
+                                 source, manifest_hash, headers):
     """Run the component analysis for given ecosystem+package+version."""
     args = {
```

Comment:
Ideally http header keys are case insensitive. Did you make sure you are able to get the values with the unnormalized keys?
.e.g `X-Request-Id == x-request-id`

Reply:
Yes.I have checked for that as well.